LogMonitor/src/LogMonitor/LogFileMonitor.cpp

```diff
@@ -2037,3 +2047,35 @@ LogFileMonitor::GetFileId(
 
     return status;
 }
+
+/// <summary>
+/// XML Formatted Log
+/// </summary>
+std::wstring LogFileMonitor::XMLFormattedLog()
+{
+    auto logFmt = L"<LogEntry><Source>File</Source><Logline>%s</Logline><FileName>%s</FileName></LogEntry>";

```

Comment:
nit: Coz of the simplicity of the two function, I'd just suggest you do away with both and just have one if/else arou:

```cpp
auto logFmt = L"<LogEntry><Source>File</Source><Logline>%s</Logline><FileName>%s</FileName></LogEntry>";
if (Utility::CompareWStrings(m_logFormat, L"JSON")) {
    logFmt = L"{\"LogEntry\": {\"Source\": \"File\",\"Logline\": \"%s\",\"FileName\": \"%s\"},\"SchemaVersion\":\"1.0.0\"}";
    // sanitize message
    Utility::SanitizeJson(fileMessage);
}
std::wstring formattedEvent = Utility::FormatString(
        logFmt,
        fileMessage.c_str(),
        fileName.c_str());
// ...
```

Reply:
of course remembering to fix the `logFmt` for JSON back to the initial string/schema.

Reply:
This has been changed